netty/src/main/java/io/grpc/transport/netty/NettyClientStream.java

```diff
@@ -128,7 +129,8 @@ protected void sendFrame(WritableBuffer frame, boolean endOfStream, boolean flus
     if (numBytes > 0) {
       // Add the bytes to outbound flow control.
       onSendingBytes(numBytes);
-      channel.write(new SendGrpcFrameCommand(this, bytebuf, endOfStream)).addListener(
+      channelWriter.enqueue(
```

Comment:
If the producer and consumer are running at the same speeds, then every frame will have a flush. That doesn't seem to be what we want to do.